addons/print/security/ir.model.access.csv

```diff
@@ -1,3 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_print_printer_all,print.printer all,model_print_printer,,1,0,0,0
+access_print_printer_all,print.printer all,model_print_printer,base.group_user,1,1,1,0
```

Comment:
Q: Why are we letting all internal users create and edit printers?

Reply:
ah good spot. I set this so I could change a barcode on it when I was debugging. I think some other access rights will have to be set, but I am not sure for which group?

Reply:
Let's do the same for print strategy too.

Reply:
Should be done now, took a while because I was having an issue with the views, which I have also pushed a change for :)